package/gluon-wlan-encryption-psk/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-wlan-encryption-psk
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-wlan-encryption-psk
+  TITLE:=Encrypt wlan using a pre-shared key (psk)
```

Comment:
Title vs. description and "pre-shared key (PSK)" and WLAN

Reply:
and the title should mention that AP and mesh are going to be crypted using that key.